payment_paypal/indico_payment_paypal/templates/event_payment_form.html

```diff
@@ -1,28 +1,84 @@
-            <input type="hidden" name="invoice" value="{{ registration.event_id }}_{{ registration.id }}">
-            <input type="hidden" name="return" value="{{ return_url }}">
-            <input type="hidden" name="cancel_return" value="{{ cancel_url }}">
-            <input type="hidden" name="notify_url" value="{{ notify_url }}">
-            <input type="image" name="submit" src="https://www.paypalobjects.com/webstatic/en_US/btn/btn_paynow_86x21.png" alt="PayPal - The safer, easier way to pay online">
-        </form>
-    </dd>
-</dl>
+{% if event_settings.itemize %}
+    <dl class="i-data-list">
+        <dl class="i-data-list">
+        <dt>{% trans %}First name{% endtrans %}</dt>
+        <dd>{{ registration.first_name }}</dd>
+        <dt>{% trans %}Last name{% endtrans %}</dt>
+        <dd>{{ registration.last_name }}</dd>
+        <dt>{% trans %}Total amount{% endtrans %}</dt>
+        <dd>{{ format_currency(amount, currency, locale=session.lang) }}</dd>
+        <dt></dt>
+ 
+        <div class="i-box-content">
+            <table class="registration-info payment-info">
+               <tr>
+                   <th class="regform-done-table-title text-left">Item </th> 
+                   <th class="regform-done-table-title">Quantity</th> 
+                   <th class="regform-done-table-title">Value</th> 
+                   <th class="regform-done-table-title">Total</th>
+               </tr>
+               {% for item in itemized_data %}
+                   <tr class="regform-done-table-item">
+                        <td class="text-left"> {{ item.title }} </td>
+                        <td class="text-right"> {{ item.quantity }} </td>
+                        <td class="text-right"> {{ item.price }} </td>
+                        <td class="text-right"> {{ item.price_total }} </td>
+                   </tr>
+               {% endfor %}
+            </table>
+        </div>
+               
+        <dt></dt>
+
+        <dd>
+            <form action="{{ settings.url }}" method="POST">
+                <input type="hidden" name="cmd" value="_cart">
+                <input type="hidden" name="upload" value="1">
+                <input type="hidden" name="charset" value="utf-8">
+                <input type="hidden" name="currency_code" value="{{ currency }}">
+                <input type="hidden" name="business" value="{{ event_settings.business }}">
+                <input type="hidden" name="invoice" value="{{ registration.event_id }}_{{ registration.id }}">
+                <input type="hidden" name="return" value="{{ return_url }}">
+                <input type="hidden" name="cancel_return" value="{{ cancel_url }}">
+                <input type="hidden" name="notify_url" value="{{ notify_url }}">
+                {% for item in itemized_data %}
+                   <input type="hidden" name="item_name_{{ item.itemNum }}" value="{{ item.title }}|truncate(127, True)">
```

Comment:
Instead of setting an `itemNum` on each item, you can use `{{ loop.index }}` or `{{ loop.index0 }}` here depending on whether you want it to start with 0 or 1.